services/core-web/src/components/noticeOfWork/NOWStatusIndicator.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable */
```

Comment:
Should we remove this and address any linting issues?

Reply:
Yes, was planning to remove once this component is live/has all the logic - This is just a starting point to have this component available for Ryan and myself.